src/Integration.UnitTests/MefServices/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -111,7 +113,7 @@ public void ActiveSolutionBoundTracker_Initialisation_Unbound()
             host.VisualStateManager.ClearBoundProject();
 
             // Act
-            using (var testSubject = new ActiveSolutionBoundTracker(this.host, this.activeSolutionTracker, loggerMock.Object))
+            using (var testSubject = new ActiveSolutionBoundTracker(this.host, this.activeSolutionTracker, loggerMock.Object, Mock.Of<IGitEvents>()))
```

Comment:
Note for the future: it would have been cleaned to refactor the existing tests to use the `CreateTestSubject`-method-with-optional-arguments pattern first.
I wouldn't do it now.

Reply:
Update: done as part of the `bound git events` PR